docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -726,3 +726,73 @@ Tasks are not running, and the server logs contain the following error message:
 
 Inline scripts are a hard requirement for Task Manager to function.
 To enable inline scripting, see the Elasticsearch documentation for {ref}/modules-scripting-security.html#allowed-script-types-setting[configuring allowed script types setting].
+
+[float]
+[[task-runat-is-in-the-past]]
+==== What do I do if the Task’s `runAt` is in the past?
+
+*Problem*:
+
+Tasks tasks property `runAt` is in the past.
+
+*Solution*:
+
+Wait a bit before declaring it as a lost cause, as Task Manager might just be falling behind on its work.
+You should take a look at the Kibana log and see what you can find that relates to Task Manager.
+In a healthy environment you should see a log line that indicates that Task Manager was successfully started when Kibana was:
+[source, txt]
+--------------------------------------------------
+server log [12:41:33.672] [info][plugins][taskManager][taskManager] TaskManager is identified by the Kibana UUID: 5b2de169-2785-441b-ae8c-186a1936b17d
+--------------------------------------------------
+
+If you see that message and no other errors that relate to Task Manager, it’s most likely that Task Manager is running fine and has simply not had the chance to pick the task up yet.
+If, on the other hand, the runAt is severely overdue, then it’s worth looking for other Task Manager or Alerting related errors, as something else may have gone wrong.
+It’s worth looking at the status field, as it might have failed, which would explain why it hasn’t been picked up or it might be running which means the task might simply be a very long running one.
+
+[float]
+[[task-marked-failed]]
+==== What do I do if the Task is marked as failed?
+
+*Problem*:
+
+Tasks marked as failed.
+
+*Solution*:
+
+Broadly speaking the Alerting framework is meant to gracefully handle the cases where a task is failing by rescheduling a fresh run in the future. If this fails to happen, then that means something has gone wrong in the underlying implementation and this isn’t expected.
+Ideally you should try and find any log lines that relate to this rule and its task, and use these to help us investigate further. Don’t forget to include the state of the rule (by using the http endpoints) in any ticket you submit.
```

Comment:
This sentence sounds like it is speaking to support instead of a user.

Reply:
Removed.